README.rst

```diff
@@ -153,6 +153,7 @@ Requires
 -  requests >= 2.13.0
 -  six
 -  retry>=0.9.2
+-  pydantic>=0.32.2
```

Comment:
you can use dataclasses module, it has a backport for python 3.5

Reply:
Sounds good, i'll start a new PR with it. Thanks!